common/src/main/java/com/microsoft/identity/common/adal/internal/AuthenticationConstants.java

```diff
@@ -569,7 +569,7 @@
         /**
          * String of ADAL version.
          */
-        public static final String ADAL_ID_VERSION = "x-client-Ver";
+        public static final String ADAL_ID_VERSION = com.microsoft.identity.common.java.AuthenticationConstants.SdkPlatformFields.VERSION;
```

Comment:
It is *really hard* to track the usage of string when they're referenced in this way.  Can we tag this reference as @Deprecated and plan to drop this static field?